src/openrct2/management/Research.cpp

```diff
@@ -342,7 +343,9 @@ void research_update()
         return;
     }
 
-    researchLevel = gResearchFundingLevel;
+    researchLevel = ((gParkFlags & PARK_FLAGS_NO_MONEY) && gResearchFundingLevel == RESEARCH_FUNDING_NONE)
```

Comment:
For this kind of branching, I prefer a classical if/else block over a ternary expression.

Reply:
Sure, will make it classical :)